KsApi/extensions/ApolloClient+RAC.swift

```diff
@@ -28,7 +28,7 @@ extension ApolloClient {
           observer.sendCompleted()
         case let .failure(error):
           print("🔴 [KsApi] ApolloClient query failure - error : \((error as NSError).description)")
-          observer.send(error: .couldNotDecodeJSON(error))
+          observer.send(error: .couldNotParseJSON)
```

Comment:
If you look at the implementation for this error envelope, the error message is an empty array. We have an extension on ErrorEnvelope on the localizedDescription property that will check if this is empty and if it is, returns 
 the localized string for "Something went wrong."

Reply:
and it's localized which is good.